src/cozmo/anim.py

```diff
@@ -89,10 +95,10 @@ def _encode(self):
             robotID=self.robot.robot_id, trigger=self.trigger.id, numLoops=self.loop_count)
 
     def _dispatch_completed_event(self, msg):
-        self.dispatch_event(EvtAnimationCompleted,
+        self._completed_event = EvtAnimationCompleted(
```

Comment:
The completed event is now stored in `self._completed_event` - this is done for all other Action types, but this one overrode `_dispatch_completed_event` and wasn't doing it. Prior to this change if you did an AnimationTrigger action, slept long enough for it to complete, and then called action.WaitForCompleted() you would receive a None object instead of the CompletedEvent which could potentially have broken client code if they were using that data for anything.